dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/configuration/ApiConfig.java

```diff
@@ -84,7 +84,7 @@ public static class PythonGatewayConfiguration {
         private int pythonPort = 25334;
         private int connectTimeout = 0;
         private int readTimeout = 0;
-        private String authToken = "jwUDzpLsNKEFER4*a8gruBH_GsAurNxU7A@Xc";
+        private String authToken = "";
```

Comment:
Removing this will result in no token verification. I think it's not a good idea.

Reply:
Restored.